api/src/app.py

```diff
@@ -27,9 +27,8 @@ def create_app() -> FastAPI:
     authenticated_routes.include_router(todo_feature.router)
     authenticated_routes.include_router(whoami_feature.router)
     app = FastAPI(title="Awesome Boilerplate", description="")
-    app.include_router(
-        authenticated_routes, prefix=prefix, dependencies=[Security(auth_with_jwt)]
-    )
+    dependencies = [Security(auth_with_jwt)] if config.AUTH_ENABLED else []
+    app.include_router(authenticated_routes, prefix=prefix, dependencies=dependencies)

```

Comment:
This will not work.
Controllers accessing the dependency injected "User" object will fail.

I will see if I can find a solution

Reply:
fixed by setting auth_scheme.auto_error = False